### ISSUE TITLE
Page introduction to be completed

### ISSUE THREAD

**Issue by nicolasdb** [Vulca/F.A.Q#2]
## First page

What do we want on the first page? 
Introduction to VULCA. 
A bit more than on vulca.eu I think. 
Push on the need to understand our Impact as maker/hacker.

The **Why** must be on this page. 
With some picture but mostly text. It's a manual, not a website. 

Then, it could be nice to have the table of content with a bit of context for each; 
_remember the TOC is also show on the left, just the titles._

**Comment by nicolasdb** [comment]
On peut utiliser un nom de domaine perso. 
![image](https://user-images.githubusercontent.com/12049360/51440266-a4c4b280-1cc5-11e9-9039-cabce8e7cd1b.png)
Mais faut préciser du DNS et des trucs...

**Comment by nicolasdb** [comment]
What could be the main chapters of this manual? 
What about the title?  **"Manual Vulca"** _mhe_

1. Introduction
1. Chapter 1
1. Chapter 2
1. Chapter 3

We can also have sub-chapters. 
We can develop them later. and collect stuff from other repo like EU maps etc... We don't need to have each time a issue on this specific repo, I mean.

**Comment by AlexandreRousselet** [comment]
![faq3](https://user-images.githubusercontent.com/45399609/51440393-798a9500-1cbe-11e9-9a22-106e094daaf2.jpg)
![faq2](https://user-images.githubusercontent.com/45399609/51440394-798a9500-1cbe-11e9-91df-7493e465b347.jpg)
![faq1](https://user-images.githubusercontent.com/45399609/51440395-798a9500-1cbe-11e9-8dfb-3073a48b2769.jpg)

**Comment by nicolasdb** [comment]
I should add some link and something to explain and guide people when they want to help. 
Maybe the link to whatsapp or the main mail, or the main entry point.

**Comment by nicolasdb** [comment]
# Disclaimer 
I think that could be important to have. At the beginning of the atlas. 
Like in a story, to setup the time and place of action, the framework of Vulca: not a network but a collective initiative, it's non politic _but work whit them_ and we are not getting paid for that _yet or some how_, blablabla and bla .
If not, people could setup their mind the wrong way during their reading of the atlas. _or the website_
 
I'm reading this article and the guy kind of start with that. 
_source: https://waitbutwhy.com/2015/06/how-tesla-will-change-your-life.html_

> Two disclaimers before we start:
> 
> 1) This is a highly politicized issue, but this post has no political agenda. I’m not political because nothing could ever possibly be more annoying than American politics. I think both parties have good points, both also have a bunch of dumb people saying dumb things, and I want nothing to do with it. So I approached this post—like I try to with every post—from a standpoint of rationality and what I think makes sense.
> 
> 2) Spoiler: The post is very pro-Tesla. Which might seem suspicious since A) Elon Musk asked me to write about this and B) I just wrote a post calling him the raddest possible man. But two things to keep in mind:
> 
> First, this isn’t commissioned by Musk, and I’m being paid $0.00 for doing it. He suggested I take the issue on because I think he thinks there’s a lack of full tree trunks in people’s heads about it—but he never suggested that I say good things about Tesla, electric cars, or anything else.
> 
> Second, the currency Wait But Why lives on is integrity. Without it, WBW loses its ability to make an impact. And integrity came first here, even at the expense of Musk potentially hating me at the end of it, if that’s what was necessary. If I didn’t think this would have made a great WBW topic, I wouldn’t have taken it on, and I’m pro-Tesla in the post because after a ton of learning and thinking—including as many counterarguments to Tesla and its worldview as I could find—that’s how I feel.
> 
> And with that, let’s dive in.

**Comment by nicolasdb** [comment]
# Atlas - définition
http://www.cnrtl.fr/definition/atlas//1

> Recueil de cartes géographiques :
> 1. Il vit la carte qu'interrogeait Monte-Cristo au moment où il était entré, et il reprit : − Vous vous occupez de géographie, Monsieur? C'est une riche étude, pour vous surtout qui, à ce qu'on assure, avez vu autant de pays qu'il y en a de gravés sur cet atlas. A. Dumas Père, Le Comte de Monte-Cristo,t. 1, 1846, p. 706.
>
